packages/server-common/test/middleware/auth-handler.test.ts

```diff
@@ -100,6 +101,26 @@ describe('Authentication middleware', () => {
     expect(response.status).toBe(401);
   });
 
+  test('Should return 500 when 500 response is returned for fetching the user', async () => {
+    const errResponse = {
+      statusCode: 500,
+      statusMessage: 'Internal server error',
+    } as unknown as Response;
+    const httpError = new HTTPError(errResponse);
+    // @ts-ignore
```

Comment:
out of curiosity - what is it moaning about here that you have to ignore?

Reply:
HTTPError.response is readonly